gotest_local_me/BBQ_test.go

```diff
@@ -0,0 +1,120 @@
+package ginkgo_me
+
+import (
+	mock_beeffarm "github.com/artiomgiza/test_me/pkgs/beef-farm/mock"
+	mock_chickenfarm "github.com/artiomgiza/test_me/pkgs/chicken-farm/mock"
+	mock_coalfarm "github.com/artiomgiza/test_me/pkgs/coal-farm/mock"
+	mock_mangalstore "github.com/artiomgiza/test_me/pkgs/mangal-store/mock"
+	"github.com/golang/mock/gomock"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_coolPriceCalculator_CalculatePrice(t *testing.T) {
+
+	t.Run("calc", func(t *testing.T) {
+		t.Run("when people counter is exceeded max value", func(t *testing.T) {
+			ctrl := gomock.NewController(t)
+			defer ctrl.Finish()
+
+			b := coolPriceCalculator{}
+			_, err := b.CalculatePrice(11)
+			assert.NotNil(t, err)
+			assert.Contains(t, err.Error(),"too much people", "should return error about people exceeded max value")
+		})
+		t.Run("when people counter is not exceeded max value",func(t *testing.T) {
+			t.Run("when get entrecote returns error", func(t *testing.T) {
+				ctrl := gomock.NewController(t)
+				defer ctrl.Finish()
+				peopleCounter := 10
+				mockBeefFarm := mock_beeffarm.NewMockProvider(ctrl)
+				mockBeefFarm.EXPECT().GetEntrecote(peopleCounter).Return(0, errors.New("test_err_7483")).Times(1)
+				t.Run("when get chicken returns error", func(t *testing.T) {
+					mockChickenFarm := mock_chickenfarm.NewMockProvider(ctrl)
+					b := coolPriceCalculator{
+						beefFarm:    mockBeefFarm,
+						chickenFarm: mockChickenFarm,
+					}
```

Comment:
initializing `coolPriceCalculator {...}` with only part of the fields is ok here, will work. 
But if test fail because of a bug that try to continue execution and call to `mockMangalStore` - nil pointer exception (because `mockMangalStore` is `nil` here )will be thrown and it will be not very clear why it happens (it can be understood of course, but require some debug).
 If there we fill the subject under test with all the mocks - it will fail on clear exception of "not call expected on the mock".